### DISCUSSION TITLE
Docker installation - symlink target needs to be mounted in container as volume

### DISCUSSION THREAD

**Discussion by hradocan** [abraunegg/onedrive#2322] [General]
Please be so kind and update https://github.com/abraunegg/onedrive/blob/master/docs/Docker.md with information provided in issue https://github.com/abraunegg/onedrive/issues/1770#issuecomment-1004552215
I've tried to setup https://github.com/abraunegg/onedrive/blob/master/docs/advanced-usage.md#upload-data-from-the-local-onedrive-folder-to-a-specific-location-on-onedrive but still have the same error: Skipping item - invalid symbolic link: ./xxxx
Now it's clear to me that was not accessible in the container.
Thank you.

**Comment by abraunegg** [comment]
@hradocan 
> Please be so kind and update https://github.com/abraunegg/onedrive/blob/master/docs/Docker.md with information provided in issue https://github.com/abraunegg/onedrive/issues/1770#issuecomment-1004552215

Feel free to create a PR with your proposed documentation changes so that you contribute back to the community.
